tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2120,11 +2120,34 @@ Status ConvertConv2DHelper(OpConverterParams* params, int group,
         {{"input_sizes", true}, {"filter", true}, {"out_backprop", false}}));
     backprop_output_size = inputs.at(0);
     tensor = inputs.at(2).tensor();
-    if (!HasStaticShape(tensor->getDimensions())) {
-      // TODO(tfeher): Allow dynamic input. We need to implement padding
-      // correction for dynamic shapes in this case.
+
+    bool has_dynamic_hw_shape{false};
+    {
+      int start_idx{0};
+      auto dims = tensor->getDimensions();
```

Comment:
I don't see how dims is used.

Reply:
It's used in two statements `if(dims.nbDims != ....)` which you recommended.